src/pages/Home/Components/LeftNavBar.tsx

```diff
@@ -32,7 +31,7 @@ function LeftNavBar() {
         borderRadius="20%"
         padding={1}
       >
-        <HamburgerIcon color="white" onClick={onOpen} />
+        <LeftDrawer />
```

Comment:
delete redundant useDisclosure. Normally you need to use this hook. But if it works without it, fine by me. :)